keps/sig-cli/3805-ssa-default/README.md

```diff
@@ -102,7 +102,8 @@ For maintainers:
   mechanisms. Removing the feature can greatly reduce the complexity of
   kubectl
 - Removes some ugly server-side code meant to deal with transition from
-  client-side apply to server-side apply.
+  client-side apply to server-side apply (see
+  https://kubernetes.io/blog/2022/10/20/advanced-server-side-apply/)
```

Comment:
Nit: this isn't associated with the last bullet in particular, so I'd just include it as its own paragraph afterwards.